prereise/gather/demanddata/transportation_electrification/smart_charging.py

```diff
@@ -0,0 +1,365 @@
+import data_helper
+import const
+import dwelling
+import numpy as np
+import scipy
+
+
+def smart_charging(
+    census_region,
+    model_year,
+    veh_range,
+    kwhmi,
+    power,
+    location_strategy,
+    veh_type,
+    filepath,
+    comm_type,
+    trip_strategy=1,
+):
+    """Smart charging function
+
+    :param int census_region: any of the 9 census regions defined by US census bureau.
+    :param int model_year: year that is being modelled/projected to, 2017, 2030, 2040, 2050.
+    :param int veh_range: 100, 200, or 300, represents how far vehicle can travel on single charge.
+    :param int kwhmi: fuel efficiency, should vary based on vehicle type and model_year.
+    :param int power: charger power, EVSE kW.
+    :param int location_strategy: where the vehicle can charge-1, 2, 3, 4, or 5; 1-home only, 2-home and work related, 3-anywhere if possibile, 4-home and school only, 5-home and work and school.
+    :param str veh_type: determine which category (LDV or LDT) to produce charging profiles for
+    :param str filepath: the path to the nhts mat file.
+    :param int comm_type: the type of Commute
```

Comment:
It would be useful to list here the allowed values

Reply:
This also has been removed since it's not needed.